src/generate.ts

```diff
@@ -6,9 +6,11 @@ const getID = (type: string): string => {
   return svgsIndex[type][Math.floor(Math.random() * svgsIndex[type].length)];
 };
 
+type Race = "Caucasian" | "African" | "Asian" | "Latino";
```

Comment:
No real reason other than it looking better to me... can you change this to: `"white" | "black" | "asian" | "hispanic"`